clients/go/cmd/zbctl/testdata/test-form-1.form

```diff
@@ -0,0 +1,26 @@
+{
+  "components": [
+    {
+      "label": "Number",
+      "type": "number",
+      "id": "Field_1ojq0w2",
+      "key": "field_16vwphu"
+    },
+    {
+      "action": "submit",
+      "label": "Submit",
+      "type": "button",
+      "id": "Field_1vctv3h",
+      "key": "field_01t07du"
+    }
+  ],
+  "type": "default",
+  "id": "Form_0w7r08e",
```

Comment:
🔧 Let's give a human-readable ID to the form. Something like: `simple_form` or something better :D

Reply:
Updated! 👍